man/ocitools-generate.1.md

```diff
@@ -137,18 +137,19 @@ read the configuration from `config.json`.
   The special *PATH* `host` removes any existing PID namespace from
   the configuration.
 
-**--poststart**=CMD
-  Path to command to run in poststart hooks. This command will be run before
-  the container process gets launched but after the container environment and
-  main process has been created.
+**--poststart**=[CMD[:ARG]]
```

Comment:
Why the brackets around the whole thing?  I'd expect:

**--poststart**=CMD[:ARG...]

or some such.  Unless we update --poststart so `--poststart=` clears any existing post-start hooks.

On the other hand, maybe not worth sinking much time into this with opencontainers/runtime-spec#483 presumably being merged soon.